src/Pages/Login/Login.js

```diff
@@ -1,17 +1,42 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 import { useHistory } from 'react-router-dom';
 import { FaRegEnvelope, FaComment } from 'react-icons/fa';
 import { IoMdClose } from 'react-icons/io';
 import { FcGoogle } from 'react-icons/fc';
 import { EMAIL_API } from '../../config';
+import { KAKAO_LOGIN_API_URL } from '../../config';
+const { Kakao } = window;
 
 const Login = props => {
+  const history = useHistory();
+
+  const kakaoLoginClickHandler = () => {
+    Kakao.Auth.login({
+      success: function (authObj) {
+        console.log(authObj);
+        console.log(Kakao.Auth.getAccessToken());
+        fetch(`${KAKAO_LOGIN_API_URL}`, {
+          method: 'POST',
+          body: JSON.stringify({
+            Authorization: authObj.access_token,
+          }),
+        })
+          .then(res => res.json())
+          .then(res => {
+            localStorage.setItem('TOKEN', res.TOKEN);
+            if (res.TOKEN) {
+              alert('GetWanted에 오신 것을 환영합니다');
+              history.push('/');
+            }
```

Comment:
- 로직상, 로컬 스토리지에 토큰을 저장한 후, 페이지 이동을 하는게 아니라,
- res.TOKEN 의 유무를 확인 한 다음에, 토큰이 있다면 로컬 스토리지에 저장을 하는 것이 더 맞을 것 같네요!
```suggestion
if (res.TOKEN) {
   localStorage.setItem('TOKEN', res.TOKEN);
   alert('GetWanted에 오신 것을 환영합니다');
   history.push('/');
}
```

Reply:
아하아하! 그러네요! 로직 순서에 맞게 수정했습니다! 감사합니다!